.github/workflows/codeowner_reviews.yml

```diff
@@ -0,0 +1,26 @@
+name: Codeowner Reviews
+
+# Controls when the workflow will run
+on: pull_request_target
+
+jobs:
+  assign-users:
+
+    runs-on: ubuntu-latest
+
+    steps:
+      # Checks-out your repository under $GITHUB_WORKSPACE, so the job can access it
+      - uses: actions/checkout@v3
+
+      #Parse the Codeowner file
+      - name: CodeOwnersParser
+        id: CodeOwnersParser
+        uses: tgstation/CodeOwnersParser@v1
+
+      #Request reviews
+      - name: Request reviews
+        if: steps.CodeOwnersParser.outputs.owners != ''
+        uses: tgstation/RequestReviewFromUser@v1
+        with:
+          separator: ' '
+          users: ${{ steps.CodeOwnersParser.outputs.owners }}
```

Comment:
Lmao I noticed we didn't have that, I just figured it was a quirk of being a downstream, hah.